Apps/Sandcastle/gallery/Globe Materials.html

```diff
@@ -0,0 +1,283 @@
+        </div>
+        <div>
+            Spacing <input style="width: 136px" type="range" min="1.0" max="500.0" step="1.0" data-bind="value: contourSpacing, valueUpdate: 'input', enable: enableContour"> <span data-bind="text: contourSpacing"></span>m
+        </div>
+        <div>
+            Line Width <input style="width: 125px" type="range" min="1.0" max="10.0" step="1.0" data-bind="value: contourThickness, valueUpdate: 'input', enable: enableContour"> <span data-bind="text: contourThickness"></span>px
+        </div>
+        <div>
+            <button type="button" data-bind="click: changeColor, enable: enableContour">Change color</button>
+        </div>
+    </div>
+</div>
+<script id="cesium_sandcastle_script">
+function startup(Cesium) {
+    'use strict';
+//Sandcastle_Begin
+var viewer = new Cesium.Viewer('cesiumContainer');
+viewer.terrainProvider = new Cesium.CesiumTerrainProvider({
+    url : 'https://assets.agi.com/stk-terrain/v1/tilesets/world/tiles',
+    requestWaterMask : true,
+    requestVertexNormals : true
+});
+viewer.scene.globe.enableLighting = true;
+
+function getElevationContourMaterial() {
+    // Creates a composite material with both elevation shading and contour lines
+    return new Cesium.Material({
+        fabric: {
+            type: 'ElevationColorContour',
+            materials: {
+                contourMaterial: {
+                    type: 'ElevationContour'
+                },
+                elevationRampMaterial: {
+                    type: 'ElevationRamp'
+                }
+            },
+            components: {
+                diffuse: 'contourMaterial.alpha == 0.0 ? elevationRampMaterial.diffuse : contourMaterial.diffuse',
+                alpha: 'max(contourMaterial.alpha, elevationRampMaterial.alpha)'
+            }
+        },
+        translucent: false
+    });
+}
+
+function getSlopeContourMaterial() {
+    // Creates a composite material with both slope shading and contour lines
+    return new Cesium.Material({
+        fabric: {
+            type: 'SlopeColorContour',
+            materials: {
+                contourMaterial: {
+                    type: 'ElevationContour'
+                },
+                slopeRampMaterial: {
+                    type: 'SlopeRamp'
+                }
+            },
+            components: {
+                diffuse: 'contourMaterial.alpha == 0.0 ? slopeRampMaterial.diffuse : contourMaterial.diffuse',
+                alpha: 'max(contourMaterial.alpha, slopeRampMaterial.alpha)'
+            }
+        },
+        translucent: false
+    });
+}
+
+var elevationRamp = [0.0, 0.045, 0.1, 0.15, 0.37, 0.54, 1.0];
+var slopeRamp = [0.0, 0.29, 0.5, Math.sqrt(2)/2, 0.87, 0.91, 1.0];
+function getColorRamp(selectedShading) {
+    var ramp = document.createElement('canvas');
+    ramp.width = 100;
+    ramp.height = 1;
+    var ctx = ramp.getContext('2d');
+
+    var values = selectedShading === 'elevation' ? elevationRamp : slopeRamp;
```

Comment:
@hpinkos can the color ramp be tweaked to be a bit less subtle for the default view?

![image](https://user-images.githubusercontent.com/782098/33220883-c6fca11c-d119-11e7-8e83-7d28cfd05bba.png)

Reply:
Or change the default view.

Reply:
I changed the default view to be of the Himalayas